cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/increment/hbase96/IncrementHandler.java

```diff
@@ -169,6 +169,9 @@ public Result preIncrement(ObserverContext<RegionCoprocessorEnvironment> e, Incr
       Result result = region.get(get);
 
       Put put = new Put(increment.getRow());
+      put.setAttribute(HBaseTable.TX_MAX_LIFETIME_MILLIS_KEY,
```

Comment:
one question here: what if that attribute is not set? Will it throw NPE?

Reply:
The javadoc for [`setAttribute()`](https://hbase.apache.org/1.1/apidocs/org/apache/hadoop/hbase/client/Attributes.html#setAttribute(java.lang.String,%20byte[])) says `In case value = null attribute is removed from the attributes map.`

Reply:
In that case, LGTM